docs/application/dotnet/guides/hot-realoading/beholder.md

```diff
@@ -0,0 +1,75 @@
+# Introduce Beholder
+
+Beholder is a hot reloading support tool for programs using Tizen.NET's NUI.
+
+Hot reloading refreshes the files that were changed without losing the state of the app.
+
+After launching the application on the emulator or real device, you can apply the changed file without stopping the application.
+
+If even in the screen that looks only go through several stages, without a step back, you can update the screen is changed files are applied
```

Comment:
I am not clear on what we are trying to say in this paragraph. Could you please check and rephrase this correctly?